packages/nodejs/.changesets/extension-now-checks-running-in-container.md

```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
+type: "change"
+---
+
+The extension is now responsible of determining if the process is running in a container. This check
+was previously made by the Node.js integration code.
```

Comment:
Not sure if this needs any changeset entries? My understanding is that this change is an implementation detail, and functionally it should behave as it did before this change. I don't know if these details about our internals are interesting information for the users. (@tombruijn thoughts?)

(If this fixes something, then we could talk in the changelog about what it fixes, i.e. "The integration now correctly detects it's running in a container when so-and-so happens")